src/iidm/VoltageLevel.cpp

```diff
@@ -220,6 +221,10 @@ VoltageLevel& VoltageLevel::setNominalVoltage(double nominalVoltage) {
     return *this;
 }
 
+void VoltageLevel::visitEquipments(TopologyVisitor& visitor) const {
+    Bus::visitEquipments<Terminal>(getTerminals(), visitor);
```

Comment:
To be discuss: even this is not exactly the same implementation, it seems strange to me that VoltageLevel relies on Bus. Maybe the visitEquipments should be a utility function of the TopologyVisitor interface